.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Chatto
+
+## Asking a Question
+
+Please don't file an issue just to ask a question.
+You'll get a faster response in [Our Gitter Room](https://gitter.im/chatto-framework/community)
+
+## Reporting a Bug
+
+### Before Submitting a Bug Report
+
+- **Make sure you're on the latest version.** If you're not on the most recent version, your problem may have been solved already. Upgrading is always the best first step.
+- **Try older versions.** If you're already on the latest release, try rolling back a few minor versions and see if the problem goes away. This will help devs narrow down when the problem first arose in the commit log.
+- **Try searching the [list of opened issues](https://github.com/badoo/Chatto/issues)** to see if the problem has already been reported. If it has, please add a comment to the existing issue instead of opening a new one.
+
+### What To Put In Your Bug Report
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title** for the issue.
+- **Describe the exact steps to reproduce the problem** in as many details as possible.
+- **Provide code samples.**
+- **Include screenshots and animated GIFs** which could clearly demonstrate the problem. You can use [this tool](https://www.cockos.com/licecap/) to capture GIFs on macOS
+- **Explain which behaviour you expect to see and why.**
+- **Provide information about your environment.**
+    - Chatto version
+    - iOS version
+    - Xcode version
```

Comment:
Maybe also include information about how Chatto is integrated into the project? Most of the people would probably use Cocoapods, but it would be worth mentioning if they use Carthage or embed Chatto manually.